user_guide/data_management_view.rst

```diff
@@ -0,0 +1,173 @@
+.. _user-guide-ui-components-view-pages:
+
+View Pages
+==========
+
+To see more details of a specific record, and to work directly with the record (create a task related to a customer, 
+appoint a calendar event for a user, turn a cart into an order, share a contact and so on), you need to get to its 
+View page.
+
+For example, this is a View page of a contact.
+
+|
+
+.. image:: ./img/data_management/view/view_01.png
+
+|
+
+On this View page you can see all the details of the record and location of the address on the map, can use the links 
+and action buttons as described in this article. 
```

Comment:
Edit: On this View page, you can see all the details of the record and its location on the map. You can use the links and action buttons as described in this article.